plugins/out_oracle_log_analytics/oci_logan_conf.c

```diff
@@ -373,16 +373,19 @@ struct flb_oci_logan *flb_oci_logan_conf_create(struct flb_output_instance *ins,
 
 
     /* Check if SSL/TLS is enabled */
-    io_flags = FLB_IO_TCP;
-    default_port = 80;
-
-#ifdef FLB_HAVE_TLS
+    #ifdef FLB_HAVE_TLS
```

Comment:
You can add a warning here but please don't change this behavior, especially not in a PR that's only meant to fix a bug, otherwise I won't be able to approve it.

Please revert this.